integration-tests/tests/analysis-context-bar.js

```diff
@@ -15,16 +15,26 @@ const testAnalysisContextBarFn = _.flow(
   // Navigate to appropriate part of UI (the analysis tab)
   await performAnalysisTabSetup(page, token, testUrl, workspaceName)
 
+  // Ensure UI displays the runtime Terminal icon is present + disabled
+  const tooltipTextTerminal = 'Terminal'
+  await findElement(page, clickable({ textContains: tooltipTextTerminal, isEnabled: false }))
+
   // Create a runtime
   await click(page, clickable({ textContains: 'Environment Configuration' }))
   await findElement(page, getAnimatedDrawer('Cloud Environment Details'))
   await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Settings' })) })
```

Comment:
I know it's not your PR but would it be good to make the search expression more specific via

```suggestion
  await noSpinnersAfter(page, { action: () => click(page, clickable({ text: 'Environment Settings' })) })
```

?

![image](https://user-images.githubusercontent.com/5438223/172485671-16b58fb9-a07c-498d-a61f-493220c3b3c6.png)

Reply:
Tried but it doesn't work. Because it's not one span element. 
<img width="239" alt="Screen Shot 2022-06-07 at 6 13 25 PM" src="https://user-images.githubusercontent.com/35533885/172492050-fb52dc82-ba03-448a-8a94-c98db9e0bcbd.png">

So now it raises a new concerns. There are 3 buttons all have `Settings` text in the label. `page.waitForXpath` always returns the first element. What if three elements have shifted in display order? I'm going to leave this as-is for now but it is a real issue with the test.

Reply:
Good point! Thinking how we could ensure the `Jupyter` button ends up being clicked on  🤔

Reply:
@breilly2 had the idea to use the `aria-label` which would [currently read 'Jupyter Environment'](https://github.com/DataBiosphere/terra-ui/blob/7355929aea1bd2401c7631d15413237df3d19520/src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js#L387). He also thought the source code should be changed to `Jupyter Environment Settings`, which I agree with. 

Cc: @cahrens @jdcanas

Reply:
@aweng98 If you could make the improvement mentioned above in this PR, great, if not, it'd be good to create an IA ticket and ping them on `#dsp-interactive-analysis`.

Reply:
I will make a follow-up PR for this change right after merge.